src/app/lib/globalStyles.test.js

```diff
@@ -6,7 +6,7 @@ describe('globalStyles', () => {
 
     await import('./globalStyles');
 
-    const injectGlobalArgs = styledComponents.injectGlobal.mock.calls[0];
+    const injectGlobalArgs = styledComponents['injectGlobal'].mock.calls[0]; // eslint-disable-line dot-notation
```

Comment:
For some reason (that I've not been able to prove) Travis will fail on the commit of the new package-lock.json if this line uses dot notation to access the `injectGlobal` key of the `styleComponents` object. 
The commit `c0790206a46fc635b0f9817a7d85f16208fa99bb` in this branch shows this as Travis fails as per the following screenshot:

![image](https://user-images.githubusercontent.com/7791726/45027334-077c2100-b039-11e8-9396-7694ff8cf46a.png)

I have a few hunches as to why Travis is failing: 
1) A dependency has changed with a fresh install which is causes a parsing error of `globalStyles.test.js` somewhere in the application prior to test running. However I would counter this saying that on my local machine I'm able to replicate the Travis error by making any line change to `globalStyles.test.js` without the package-lock change. 
2) Somewhere in the minification process of the application there is an error being caused which is not being surfaced to the failing test. I would counter this by saying on different local machine this test seems to pass or fail regardless of new line additions.
3) The test is passing for Travis on latest and for my local with the package-lock changes due to the minification process seeing no change to the file and uses a cached version of the file. This could explain the issue as locally the test would pass or fail with line changes varying on the users local node_modules cache. With node_modules being installed fresh on Travis we therefore would see this issue on ci and not on certain local machine. It would also explain how locally with node_modules removed and fresh install how any change to the test file would update the minification cache and reveal the same error as seen on Travis